activejob/lib/active_job/railtie.rb

```diff
@@ -25,16 +25,29 @@ class Railtie < Rails::Railtie # :nodoc:
       options = app.config.active_job
       options.queue_adapter ||= :async
 
+      options.after_initialize do
```

Comment:
Should not this be `app.config`? And why do it twice?

Reply:
@byroot recommended [copying the approach from `ActiveRecord::Railtie`](https://github.com/rails/rails/blob/bfa3a5baabec97855d77615645bb6757f19a4502/activerecord/lib/active_record/railtie.rb#L223-L268) for consistency, which is why we're doing it twice. I'm glad to change this to whatever is determined to be the correct way to do it.

However, I do see that I misread, and `ActiveRecord::Railtie` is calling `after_initialize` on `config`, not `configs`, so I'll fix that. It is clearer here with the (existing) name `options`, which stands out as different.